### ISSUE TITLE
[Request] Signed binary tarball as part of new releases

### ISSUE THREAD

**Issue by lead4good** [gramineproject/gramine#235]
I am currently working on creating a buildpack for gramine, for this work it would be nice to have a signed binary tarball provided via the github releases. Would that be possible?

**Comment by mkow** [comment]
Yup, we have this in plans (together with reproducible builds), but we'll see if we manage to set everything up for the next release.
